fk/luae_na/src/main/java/se/inera/intyg/common/luae_na/v1/model/converter/RespConstants.java

```diff
@@ -21,6 +21,42 @@
 
 public class RespConstants {
 
+    public static final String DIAGNOS_CATEGORY_ID = "diagnos";
+    public static final String DIAGNOS_SVAR_ID_6 = "6";
+    public static final String DIAGNOS_BESKRIVNING_DELSVAR_ID_6 = "6.1";
+    public static final String DIAGNOS_DELSVAR_ID_6 = "6.2";
+    public static final String BIDIAGNOS_1_BESKRIVNING_DELSVAR_ID_6 = "6.3";
+    public static final String BIDIAGNOS_1_DELSVAR_ID_6 = "6.4";
+    public static final String BIDIAGNOS_2_BESKRIVNING_DELSVAR_ID_6 = "6.5";
+    public static final String BIDIAGNOS_2_DELSVAR_ID_6 = "6.6";
+    public static final String DIAGNOS_SVAR_JSON_ID_6 = "diagnoser";
+    public static final String DIAGNOS_CATEGORY_TEXT = "KAT_3.RBK";
+    public static final String DIAGNOS_SVAR_TEXT = "FRG_6.RBK";
+    public static final String DIAGNOS_SVAR_BESKRIVNING = "FRG_6.HLP";
+    public static final String DIAGNOS_DELSVAR_TEXT = "DFR_6.2.RBK";
+    public static final String DIAGNOS_ICD_10_ID = "ICD_10_SE";
+    public static final String DIAGNOS_ICD_10_LABEL = "ICD-10-SE";
+    public static final String DIAGNOS_KSH_97_ID = "KSH_97_P";
+    public static final String DIAGNOS_KSH_97_LABEL = "KSH97-P (Primärvård)";
+
+
+    public static final String DIAGNOSGRUND_SVAR_ID_7 = "7";
+    public static final String DIAGNOSGRUND_TEXT = "FRG_7.RBK";
+    public static final String DIAGNOSGRUND_DELSVAR_ID_7 = "7.1";
+    public static final String NYDIAGNOS_SVAR_ID_45 = "45";
+    public static final String DIAGNOSGRUND_NYBEDOMNING_DELSVAR_ID_45 = "45.1";
+    public static final String DIAGNOS_FOR_NY_BEDOMNING_DELSVAR_ID_45 = "45.2";
+
+    public static final String DIAGNOSGRUND_NYBEDOMNING_SVAR_TEXT = "FRG_45.RBK";
+    public static final String DIAGNOSGRUND_NYBEDOMNING_SVAR_TEXT_JA = "FRG_45.1.1.RBK";
```

Comment:
Have to be removed as they don't exists in the text-files for the version. Both TEXT-IDS for JA and NEJ.

Reply:
Fixed!